source/evtpatch.cpp

```diff
@@ -206,13 +206,25 @@ void insertTrampolineCall(EvtScriptCode* ptr, EvtScriptCode* script) {
     ptr[TRAMPOLINE_CALL_EVT_OFFSET] = reinterpret_cast<s32>(script);
 }
 
+/// @brief Jumps execution of an EVT entry to a specified location
+/// @param ptr The place to write the instruction at
+/// @param scriptCode The EvtScriptCode to replace the destination with
+/// @param dst The size of dst
+void insertScriptCall(EvtScriptCode* ptr, EvtScriptCode* scriptCode, s32 size) {
+    //wii::os::OSReport("%x %x %x %x\n", (unsigned char)ptr[0], (unsigned char)ptr[1], (unsigned char)ptr[2], (unsigned char)ptr[3]);
+    //wii::os::OSReport("%x %x %x %x\n", (unsigned char)scriptCode[0], (unsigned char)scriptCode[1], (unsigned char)scriptCode[2], (unsigned char)scriptCode[3]);
+    msl::string::memcpy(ptr, scriptCode, size);
+    //wii::os::OSReport("%x %x %x %x\n", (unsigned char)ptr[0], (unsigned char)ptr[1], (unsigned char)ptr[2], (unsigned char)ptr[3]);
+}
```

Comment:
Can this function only insert call opcodes? Please keep function names accurate. Furthermore, this function is not in the header file. If this is intentional, mark this function as static.